tests/core/dry/test.sh

```diff
@@ -3,9 +3,9 @@
 
 rlJournalStart
     rlPhaseStartTest
-        rlRun "tmt run --dry"
-        rlRun "tmt run --dry -dv"
-        rlRun "tmt run --dry -ddvv"
-        rlRun "tmt run --dry -dddvvv"
+        rlRun "tmt run --dry" 3
```

Comment:
I understand why it reports exit code 3 but I object. As a user I asked it to not run anything and tmt satisfied my requrest, but exitcode is non-zero which indicates error (usually)

Reply:
Good point, adjusted in 7394537.